mysql_init.sls

```diff
@@ -0,0 +1,95 @@
+install_mysql:
+  pkg.installed:
+    - pkgs:
+      - mysql-server
+      - python-mysqldb 
+
+reload-mysql:
+  service.running:
+    - name: mysql
+    - enable: True
+    - reload: True
+
+root_user:
+  mysql_user.present:
+    - name: 'root'
+    - password: 'TOPSECRET'
+
+mysql remove anonymous users:
+  mysql_user.absent:
+    - name: ''
+    - host: 'localhost'
+    - connection_user: 'root'
+    - connection_pass: 'TOPSECRET'
+    - connection_charset: utf8
+
+mysql remove test database:
+  mysql_database.absent:
+    - name: test
+    - host: 'localhost'
+    - connection_user: 'root'
+    - connection_pass: 'TOPSECRET'
+    - connection_charset: utf8
+
+create api database:
+  mysql_database.present:
+    - name: api_production
+    - host: localhost
+    - connection_user: 'root'
+    - connection_pass: 'TOPSECRET'
+    - connection_charset: utf8
+
+create obs for api user@localhost:
+  mysql_user.present:
+    - name: obs
+    - host: localhost
+    - password: 'topsecretpasskey'
+
+create obs for api user@%:
+  mysql_user.present:
+    - name: obs
+    - host: \%
+    - password: 'topsecretpasskey'
+
+obs_grants to api obs@localhost:
+  mysql_grants.present:
+    - host: localhost
+    - database: api_production.\*
+    - grant: all privileges
+    - user: obs
+
+obs_grants to api obs@%:
+  mysql_grants.present:
+    - host: \%
+    - database: api_production.\*
+    - grant: all privileges
+    - user: obs
+
+create webui database:
+  mysql_database.present:
+    - name: webui_production
+    - host: localhost
+    - connection_user: 'root'
+    - connection_pass: 'TOPSECRET'
+    - connection_charset: utf8
+
+#Grant permissions to obs:
```

Comment:
Try to avoid commented code :)
(or explain why you commented it)

Reply:
I kept that as I wanted you to review that part once :) it didn't worked the way I thought it would so I finally wrote a sh file. I'll definitely remove it.